pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
@pmorie There is duplication between gce_pd and the new proposed plugins for NFS and iCSCI, specifically around methods like ```isMountPoint```.    A refactoring in this area would help those other plugin authors.